source/basic.tex

```diff
@@ -4845,11 +4845,11 @@
 ``\tcode{\keyword{unsigned} \keyword{char}}'', ``\tcode{\keyword{unsigned} \keyword{short} \keyword{int}}'',
 ``\tcode{\keyword{unsigned} \keyword{int}}'', ``\tcode{\keyword{unsigned} \keyword{long} \keyword{int}}'', and
 ``\tcode{\keyword{unsigned} \keyword{long} \keyword{long} \keyword{int}}''.
-\indextext{type!extended unsigned integer}%
+\indextext{extended unsigned integer type|see{type, extended unsigned integer}}%
 Likewise, for each of the extended signed integer types,
-there exists a corresponding \defn{extended unsigned integer type}.
+there exists a corresponding \defnx{extended unsigned integer type}{type!extended unsigned integer}.
```

Comment:
Make this `\defnadj{extended unsigned integer}{type}` and you get the "see type, extended unsigned integer" entry for free (the other line can go).